jsonrpc/eth_endpoint_test.go

```diff
@@ -649,8 +663,8 @@ func TestEth_State_GetStorageAt(t *testing.T) {
 				}
 			}
 			dispatcher := newTestDispatcher(hclog.NewNullLogger(), store)
-			blockNumber, _ := createBlockNumberPointer(tt.blockNumber)
-			res, err := dispatcher.endpoints.Eth.GetStorageAt(tt.address, tt.index, blockNumber)
+			// blockNumber, _ := createBlockNumberPointer(tt.blockNumber)
```

Comment:
Please clean up this comment

Reply:
Nice catch ! Removed in https://github.com/0xPolygon/polygon-sdk/pull/284/commits/78906462560351619f9a53584b93f252d2283ccc.